_includes/configuration/images-dataimg-filter.md

```diff
@@ -10,7 +12,9 @@ The **images_dataimg_filter** option lets you specify a filter predicate functio
 tinymce.init({
   selector: 'textarea',  // change this value according to your HTML
   images_dataimg_filter: function(img) {
-    return img.hasAttribute('internal-blob');
+    return !img.hasAttribute('internal-blob');  // blocks the upload of <img> elements with the attribute "internal-blob".
```

Comment:
I'm surprised, I would have thought the opposite would happen based on the return value (true/false). Is it worth saying "if the callback returns false, upload will be blocked" or something? To be more explicit

Reply:
Yeah, I expected the same originally tbh

Reply:
extended the option description to clarify: 24e41345a1a0bf9ac32e45fb263a273ce6846b65